examples/kafka-example/app.js

```diff
@@ -12,12 +12,21 @@ orka({
   logoPath: __dirname + '/logo.txt',
   beforeStart: async () => {
     const config = require('./config');
-    const topic = config.kafka.consumer.topics.name;
-    // Will copy offsets from oldGroupId to the new one
-    await getKafka().renameGroupId([{ groupId: config.kafka.groupId, topic, oldGroupId: config.kafka.oldGroupId }]);
 
-    const KafkaHandler = require('./handler');
-    new KafkaHandler(getKafka(), { topic, fromBeginning: true });
+    if (config.nodeEnv != 'test') {
```

Comment:
Because we use example app for health e2e test. And if we create a consumer then test doesn't close grasefully even if we disconnect producer and close the server

Reply:
Thanks for the explanation. It is difficult to disconnect the consume because you don't have the instance right?

Reply:
Exactly